lib/Mojo/Server.pm

```diff
@@ -8,8 +8,9 @@ use Mojo::Util qw(md5_sum);
 use POSIX ();
 use Scalar::Util qw(blessed);
 
-has app           => sub { shift->build_app('Mojo::HelloWorld') };
-has reverse_proxy => sub { $ENV{MOJO_REVERSE_PROXY} };
+has app             => sub { shift->build_app('Mojo::HelloWorld') };
+has reverse_proxy   => sub { $ENV{MOJO_REVERSE_PROXY} || @{shift->trusted_proxies} };
```

Comment:
should this be `sub { $ENV{MOJO_REVERSE_PROXY} || !!@{shift->trusted_proxies} };`